code/modules/goals/station_goals/bluespace_tap.dm

```diff
@@ -87,8 +87,6 @@
 		/obj/item/grenade/clusterbuster/cleaner,
 		/obj/item/grenade/clusterbuster/soap,
 		/obj/item/toy/katana,
-		/obj/item/stack/sheet/metal/twenty,
```

Comment:
Document change, (and explain why??)

Reply:
Well since it output mats now so i removed unnecessary metal sheets